spec/javascript/orangelight/availability.spec.js

```diff
@@ -151,6 +151,23 @@ describe('AvailabilityUpdater', function() {
     spy.mockRestore()
   })
 
+  // Make sure that the code to handle undetermined availability status updates
```

Comment:
Notice that this test is pretty shallow, it only makes sure the HTML was updated when we directly call the `update_single_undetermined` method. 

To test the logic that makes that call in the code, we would need to mock the an Ajax call that returns an HTTP 429 on the first try and an HTTP 200 on the second try. That seems more complicated than I think we want to pursue.

Reply:
@hectorcorrea do you want to open a new issue for that ^ ?

Reply:
@christinach done: https://github.com/pulibrary/orangelight/issues/2498

Reply:
Thank you @hectorcorrea